scripts/exampleSetup.ts

```diff
@@ -119,97 +88,59 @@ async function main() {
   await sdl.approve(veSDL.address, MAX_UINT256)
 
   // Create max lock with 10M SDL
-  const create_lock_gas_estimate = await veSDL.estimateGas.create_lock(
-    BIG_NUMBER_1E18.mul(10_000_000),
-    (await getCurrentBlockTimestamp()) + 4 * YEAR,
-  )
   await veSDL.create_lock(
     BIG_NUMBER_1E18.mul(10_000_000),
     (await getCurrentBlockTimestamp()) + 4 * YEAR,
-    { gasLimit: create_lock_gas_estimate },
+    { gasLimit: 3_000_000 },
   )
   console.log(
     "(10M SDL/4 Year) lock created at timestamp: ",
     await getCurrentBlockTimestamp(),
   )
 
-  // Force mine 1 block and then skip timestamp to specified time
-  await setTimestamp((await getCurrentBlockTimestamp()) + 2 * YEAR)
+  // Only manually trigger mining parameters if rate is 0 (uninitialized)
+  const rewardRate = await minter.rate()
+  if (rewardRate.eq(0)) {
+    await minter.update_mining_parameters()
+  }
 
-  await setTimestamp(
-    Math.floor(((await getCurrentBlockTimestamp()) + WEEK) / WEEK) * WEEK,
-  )
-  await minter.update_mining_parameters()
-
-  let get_gauge_weight_gas_estimate =
-    await gaugeController.estimateGas.get_gauge_weight(gauge_addresses[0])
-  let get_relative_gauge_weight_gas_estimate =
-    await gaugeController.estimateGas["gauge_relative_weight(address)"](
-      gauge_addresses[0],
-    )
-  let i
-  const change_gauge_weight_gas_estimate =
-    await gaugeController.estimateGas.change_gauge_weight(
-      gauge_addresses[0],
-      10000,
-    )
-  for (i = 0; i < gauge_names.length; i++) {
-    console.log(
-      gauge_names[i],
-      "gauge_weight: ",
-      (
-        await gaugeController.get_gauge_weight(gauge_addresses[i], {
-          gasLimit: get_gauge_weight_gas_estimate,
-        })
-      ).toString(),
-    )
-    console.log(
-      gauge_names[i],
-      "relative_weights: ",
-      (
-        await gaugeController["gauge_relative_weight(address)"](
-          gauge_addresses[i],
-          { gasLimit: get_relative_gauge_weight_gas_estimate },
-        )
-      ).toString(),
-    )
```

Comment:
Console.log is bit hard to understand here. Better to save string as variables then construct the console.log line so that we can easily see what is being written to console.